packages/expo-secure-store/ios/EXSecureStore/EXSecureStore.m

```diff
@@ -178,7 +196,7 @@ + (NSString *) _messageForError:(NSError *)error
       return @"Unable to decode the provided data.";
 
     case errSecAuthFailed:
-      return @"The user name or passphrase you entered is not correct.";
+      return @"Authentication failed. It is possible that the device has no screen lock.";
```

Comment:
I'm not sure why you're changing the message here? 🤔
What is `screen lock`? 🤔 
My understanding is:
- previous message: `you have made a mistake typing in the passphrase (or device PIN maybe?)`
- your change: `There's no authentication method configured for this device`.

Is it possible to distinguish between these two cases? If no I would go with `Authentication failed. Provided passphrase/PIN is incorrect or there is no user authentication method configured for this device.`

Reply:
[I went with changing the error message](44e98a6e8f61e6904d1a1e73936a0c963eae7e2d) as there are no separate error codes for these cases.